### ISSUE TITLE
[Accessibility Checker] Production License Keys - Site Onboarding Wizard

### ISSUE THREAD

**Issue by shascher** [Good-Bids/goodbids#831]
Hi @jaspercroome - Dropping this overview here for the current Accessibility Checker plugin in GitHub as there will need to be some development tasks based on either of the options below.

### Overview
Currently, on Production, we are running into an issue with licensing that we did not experience on our Staging and Development environments.

During testing of the plugin, our multi-site setup worked based on the way we installed and coded the plugin to work across our network sites. However, when porting this to Production, we identified that the a single license actually does not work across the multi-site setup process due to the way Accessibility Checker activates plugin licenses on Production. This is causing the hiccup in the onboarding step for activating Accessibility Checker.

I gave Equalize Digital a call yesterday and talked with chris@equalizedigital.com. Based on that conversation, I got more information on how each network site in our setup actually requires its own separate license on Production.

This is a different understanding and approach than what we had during our initial plugin testing based on how it was working on our non-production environments. Chris recommended that if there are only a few pages on the sites, and if the custom post type is not strictly templated, the free version of the plugin would work fine for our needs. 

It all depends on how a Nonprofit might continue to use the site outside of the initial site creation and Auctions. Below are my thoughts on possible paths forward.

### Pivot Options

1. The quickest and most streamlined option is to purchase additional licenses with Accessibility Checker: https://equalizedigital.com/accessibility-checker/pricing/. This may be cost prohibitive, but depending on how many sites you will have at launch, or in the foreseeable future, likely the quickest solution.

2. Remove the Pro plugin from the Nonprofit sites and remove the onboarding step for Accessibility Checker in the onboarding wizard. This will allow Nonprofit sites to check all pages for accessibility except for the Auction pages. Because the auction pages are very templated, or it appears they will be, they currently pass accessibility standards and as long as new blocks are not added, there would not be additional a11y checks required.

Happy to hop on a call and let me know your thoughts!
